### ISSUE TITLE
ConcurrentTileCache does not seem to apply correct memory purging

### ISSUE THREAD

**Issue by aaime** [geosolutions-it/jai-ext#105]
Got a OOM in a system that used it, checking the memory dump, it contained 8 million CachedTileImpl, which in turn were holding over 1.3GB only as BigInteger keys... there is something wrong with cache cleaning there, but not sure what...

**Comment by simboss** [comment]
Do we use it anywhere any longer?

**Comment by aaime** [comment]
Nope, we switched to ConcurrentTileCacheMultiMap and haven't used it for a long time.

**Comment by simboss** [comment]
Then I would close this one and start a deprecation cycle before next release.
